packages/ag-charts-enterprise/src/animation/animation.ts

```diff
@@ -12,16 +12,6 @@ export class Animation extends _ModuleSupport.BaseModuleInstance implements _Mod
     @Validate(BOOLEAN)
     public enabled = true;
 
-    @ActionOnSet<Animation>({
-        newValue(value: number | undefined) {
-            if (!this.animationManager) return;
-            this.animationManager.defaultOptions.duration = value;
-            this.animationManager.skipAnimations = value === 0;
-        },
-    })
-    @Validate(NUMBER(0))
-    public duration?: number;
```

Comment:
This may still be needed?

Reply:
no one seems to be using it and it looks to me like a hack from the past

Reply:
This is definitely still needed. It passes the user's configuration of `duration` through to the animation manager. Wherever we call `this.ctx.animationManager.defaultDuration()` in series, it uses this value.